src/components/Header.js

```diff
@@ -1,19 +1,39 @@
 import Link from 'next/link';
-import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+const NaviHeader = styled.header`
+	display: flex;
+	padding: 15px;
+	background-color: #f01878;
+	color: white;
+
+	& a {
+		padding: 0 100px;
+		color: white;
+		text-decoration: none;
+	}
+	& a:hover {
+		color: blanck;
+	}
+	& a:active {
+		color: #4878ff;
+	}
+	& a:visited {
+		color: white;
+	}
+`;
 
 export default function Header() {
-	const router = useRouter();
 	return (
-		<header>
+		<NaviHeader>
 			<nav>
 				<Link href="/">
-					<a>Home</a>
+					<a>Asana Home</a>
 				</Link>
```

Comment:
In order to style your active link differently, you would need to give the `Link` a prop called 'active' if the current URL matches the path of the `Link` component:
```jsx
const router = useRouter();

return (
<nav>
    <Link href="/">
        <StyledLink active={router.pathname === '/'}>Daily entry</StyledLink>
    </Link>
</nav>
```

Please do not forget to import `useRouter` 🙂